RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+CreateEntity.swift

```diff
@@ -139,7 +139,7 @@ extension OnLedgerEntitiesClient {
 	) async throws -> OnLedgerEntity.Validator? {
 		@Dependency(\.gatewaysClient) var gatewaysClient
 		let networkId = await gatewaysClient.getCurrentNetworkID()
-		let xrdAddress = knownAddresses(networkId: networkId.rawValue).resourceAddresses.xrd.addressString()
+		let xrdAddress = Sargon.xrdAddressOfNetwork(networkId: networkId)
```

Comment:
We only used the collection of known addresses to get XRD ResourceAddress, so we use expose API for that.